docs/source/rules.rst

```diff
@@ -9,6 +9,26 @@ a particular rule or set of rules. The intent is that the definition of
 each specific rule should be really streamlined and only contain the logic
 for the rule itself, with all the other mechanics abstracted away.
 
+Core Rules
+__________
+
+Certain rules are belong to the :code:`core` rule group. In order for
+a rule to be designated as :code:`core`, it must meet the following
+criteria:
+
+* Stable
+* Applies to most dialects
+* Could detect a parsing issue
```

Comment:
Ah I meant 'syntax error'

Something like this

```sql
select
    x,
    y
    z
from my_table
```

This is 'valid' SQL, but very likely the user forgot a comma

This would warn about implicit aliasing, which would point to the  larger syntax oversight 

I can change the language I used in the docs